src/main/java/gift/controller/AdminController.java

```diff
@@ -0,0 +1,72 @@
+package gift.controller;
+
+
+import gift.model.Gift;
+import gift.model.GiftDao;
+import gift.service.GiftService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+
+@Controller
+public class AdminController {
+    private GiftService giftService;
+
+    AdminController(GiftService giftService){
+        this.giftService = giftService;
+    }
+
+    @GetMapping
+    public String index(){
+        return "redirect:/admin";
+    }
+
+    @GetMapping("/admin")
+    public String adminHome(Model model){
+        Collection<Gift> giftlist = giftService.getAllGifts();
+        model.addAttribute("giftlist",giftlist);
+        return "admin";
+    }
+
+    @GetMapping("/admin/create")
```

Comment:
추후에 gift 말고 다른 것들도 생긴다면 무엇에 대한 생성 요청인지 구분이 어려울 것 같아요! 
따라서 다음과 같이 prefix를 넣어주는 것이 좋을 것 같습니다ㅎㅎ
```suggestion
    @GetMapping("/admin/gift/create")
```

Reply:
관리자페이지에서 다른 데이터에 대한 생성 작업도 할 수 있다는 걸 놓친 것 같습니다. 감사합니다.